lib/msf/core/exploit/java/serialized/rmi/runner.rb

```diff
@@ -0,0 +1,86 @@
+# -*- coding: binary -*-
+
+module Msf
+  module Exploit::Java::Serialized::RMI
+
+    class RMICheck
+      attr_reader :jmxprober
+
+      def initialize(host, port, ssl: false, username: nil, password: nil, rc: RunConfig.new)
+        @host = host
+        @port = port
+        @ssl = ssl
+        @jmxcreds = nil
+        if !username.nil? && !password.nil?
+          @jmxcreds = Rex::Java::Serialization::Metamodel::JavaArray.new('Ljava/lang/String;', 
+                                                                         %w[test test])
+        end
+
+        @rc = rc
+
+        @reg = Rex::Java::Serialization::Metamodel::Registry.new()
+        @reg.load('model/base-java9.json')
+        @reg.load('model/rmi.json')
+      end
+
+      def run
+        prober = RMIProber.new(@host, @port, @reg, ssl: @ssl, rc: @rc)
+        vectors = prober.run
+
+        if @rc.checkRefs?
```

Comment:
It looks like the indentation got messed up somewhere in this conditional.

It's hard to read due to the messed up indentation, but it looks like all of the code exists inside this conditional (?), in which case you could simply bail out early instead.

```ruby
return unless @rc.checkRefs?
```